code/modules/power/rtg.dm

```diff
@@ -102,5 +103,51 @@
 	power_gen = 20000
 	circuit = null
 
-/obj/machinery/power/rtg/debug/RefreshParts()
```

Comment:
Why was this removed? Calling refereshparts would set the debug rtg's power to zero.